manifests/renew.pp

```diff
@@ -84,4 +91,23 @@
     minute   => $cron_minute,
     monthday => $cron_monthday,
   }
+
+  if ($disable_distro_cron) {
+    if ($facts['service_provider'] == 'systemd') {
+      if $distro_renew_timer {
```

Comment:
I'd recommend changing this to be ANDed condition to avoid 3 level if condition.
```suggestion
  if $disable_distro_cron and $distro_renew_timer and $facts['service_provider'] == 'systemd' {
```

Would need to adjust depth of resources and remove extra closing brackets.